src/invidious/routes/live.cr

```diff
@@ -0,0 +1,34 @@
+module Invidious::Routes::Live
```

Comment:
Same here: this route is related to channels, so it should go under `Invidious::Routes::Channels`